Luminary099/R31.agc

```diff
@@ -169,17 +169,17 @@ COMPDISP	VLOAD	VSU
 			UNITZ
 		CALL
 			CDU*NBSM
-		VXM	PUSH		# UNIT(Z)/4 TO PD 6-11
+		VXM	PUSH		# UNIT (Z)/4 TO PD 6-11
 			REFSMMAT
-		VPROJ	VSL2		# UNIT(P)=UNIT(UZ-(UZ)PROJ(UR))
+		VPROJ	VSL2		# UNIT(P)=UNIT(UZ -(UZ)PROJ(UR))
 			0D
 		BVSU	UNIT
 			6D
 		PDVL	VXV		# UNIT(P) TO PD 12-17
 			0D		# UNIT(RL)
 			VONE
 # Page 706
-		VXV	DOT		# (UR * VL) * UR . U(P)
+		VXV	DOT		# (UR * VL)*UR . U(P)
 			0D
 			12D
 		PDVL			# SIGN TO 12-13, LOAD U(P)
```

Comment:
8 lines below, change 

```
#	RTHETA = 1 - RTHETA
``` 

to (= aligns with **N** of **N**eg on line above)

```
#   RTHETA = 1 - RTHETA
```